components/selection/selection-input.js

```diff
@@ -56,6 +58,14 @@ class Input extends SkeletonMixin(LabelledMixin(LitElement)) {
 			:host([hidden]) {
 				display: none;
 			}
+
+			.d2l-input-radio-disabled-tooltip.d2l-disabled:hover,
+			.d2l-input-radio-disabled-tooltip.d2l-disabled.d2l-hovering,
+			.d2l-input-radio-disabled-tooltip.d2l-disabled:${unsafeCSS(getFocusPseudoClass())} {
+				border-color: var(--d2l-color-celestine) !important;
```

Comment:
`!important` always feels wrong, and it's creating a disconnect between `radioStyles` and this implementation. For example, `radioStyles` could change something (e.g. add a background colour) and this code wouldn't know about it.

Instead, could we move this functionality into `radioStyles` so that it's in control of everything?